beaker.gemspec

```diff
@@ -47,6 +47,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'open_uri_redirections', '~> 0.2.1'
   s.add_runtime_dependency 'in-parallel', '~> 0.1'
   s.add_runtime_dependency 'thor', ['>= 1.0.1', '< 2.0']
+  s.add_runtime_dependency 'psych'
```

Comment:
This is a bad idea. It will pull in the latest Psych on older Rubies as well instead of using the vendored gem inside Ruby itself.

Instead, I'd update `Gemfile` to use `gem 'rdoc', '< 6.4' if RUBY_VERSION < '3.1'`